common.mk

```diff
@@ -12,16 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-include $(ROOT_DIR_RELATIVE)/versions.mk
+include $(REPO_ROOT)/versions.mk
 
 # Ensure Make is run with bash shell as some syntax below is bash-specific
 SHELL:=bash
 .ONESHELL:
-.SHELLFLAGS := -eu -o pipefail -c
+# .SHELLFLAGS := -eu -o pipefail -c
```

Comment:
Somehow, these SHELLFLAGS being set was leading to failures every time this and other Makefiles would call the `shell` command. I do not fully understand why, but without them it works.

Reply:
[explanation of shell flags](https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425)

-e exit if a failure occurs instead of chugging along
-u exit if an undefined variable is expanded/used
-pipefail exit if one command piped into another fails, so `false | true` should exit. This doesn't mean that `false || true` does though.

-c typically is used with a string thereafter so the shell executes the string as a command once it starts.

I'm not sure it's needed here, or when it was added, but maybe it is.

Did you try removing one flag at a time to see which flag in particular was causing the issue?

I don't think we should just blanket comment them all out.

Reply:
I agree. I've also spoken with @jweite-amazon and noted that these failures are OS specific

Reply:
Thanks everybody, moving to using `-euc` instead to avoid spaces, which are not handled properly in the linux bash.